netContain/contivInit.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#Initialize contiv Net Plugin and Net Master as required
+# Values below need to be tailored as per needs
+
+source contivRc
+
+if [ $IS_NETMASTER == 1 ]
+then
+   /contiv/bin/netmaster 2> /var/log/contiv/netmaster.errlog 1> /var/log/contiv/netmaster.log &
```

Comment:
Can we add a cluster-store option to netmaster so that we can point to either etcd or consul?